trust-center-agent/src/main/java/care/smith/fts/tca/deidentification/FhirMappingProvider.java

```diff
@@ -74,6 +75,16 @@ public Mono<TransportMappingResponse> generateTransportMapping(TransportMappingR
         .map(cdShift -> new TransportMappingResponse(transferId, transportMapping, cdShift));
   }
 
+  private Mono<Tuple3<String, String, String>> fetchPseudonymAndSalts(
+      String patientId, TCADomains domains, Duration maxDateShift) {
```

Comment:
Do we use 2 or 4 spaces indent in the project?

Reply:
We are using google-java-format. 2 for normal indentation 4 for line continuation.

Reply:
Ah ok, I see.